image_op/norm_by_background_mean.m

```diff
@@ -0,0 +1,22 @@
+function M_norm = norm_by_background_mean(movie)
+%Normalizes every frame in the movie by the mean pixel value of the
+%background to mitigate the temporal variation in the brightness
+%
+%   movie: movie matrix , [h x w x num_frames]
+%
+% Hakan Inan, 15-Jan-4
+%
+
+M_norm = zeros(size(movie),'single');
+[height,width,numFrames] = size(movie);
+
+for k = 1:numFrames
+    if (mod(k,1000)==0)
+        fprintf('  Frames %d of %d done\n', k, numFrames);
+    end
+    Frame = reshape(movie(:,:,k),height*width,1);
+    thresh = quantile(Frame,0.7);
```

Comment:
The quantile parameter (`0.7`) should be exposed as a function parameter (e.g. `quantile_threshold`). Also, when `quantile_threshold == 1` then you should skip the `quantile` operation and perform a naive mean without regards to histogram.